generic/tests/ntpd.py

```diff
@@ -60,10 +62,10 @@ def server_config(self):
         2.start ntpd service;
         3.restrict the host and guest
         """
-        logging.info("waiting for login server.....")
+        LOG_JOB.info("waiting for login server.....")
```

Comment:
All `LOG_JOB` inside this script are safe to be replaced with `self.test`, which is also `avocado.core.test.Test` type.

Reply:
Will update this and the next ones.